cluster/gce/gci/configure-helper.sh

```diff
@@ -329,6 +343,7 @@ function start-kubelet {
       flags+=" --register-schedulable=false"
       flags+=" --reconcile-cidr=false"
       flags+=" --pod-cidr=10.123.45.0/30"
+      reconcile_cidr=false
```

Comment:
Ah I made a mistake that I thought the default value is false. Just check it, this flag is true by default. I will update the code